packages/react-components/src/components/knowledge-graph/KnowledgeGraphPanel.tsx

```diff
@@ -172,7 +177,7 @@ export const KnowledgeGraphContainer: React.FC<KnowledgeGraphInterface> = ({
         ? onGraphResultChange([...nodeData.values()], [...edgeData.values()])
         : onGraphResultChange([...nodeData.values()]);
     }
-    cy.current?.fit();
+    if (nodeData.size > 1) cy.current?.fit();
```

Comment:
Wouldn't this be `>==` or is being equal to 1 something you would want to ignore?

Reply:
yup equal to 1 we want to ignore, the UX felt when we have a one node result it fits the whole screen and it looks big and ugly, so instead we don't want it to fit the whole canvas.